server/models/dataset_helpers/image_metadata.py

```diff
@@ -21,6 +21,19 @@
 import six
 
 
+_melanocyticDiagnoses = [
```

Comment:
Can you put this as a class object under `MelanocyticFieldParser`? As we get more of these, it will be messy to have lots of global objects floating around.